pages/Loginin.html

```diff
@@ -0,0 +1,122 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login page | Brushsy-mart | Online Shopping site for makeup brushes.</title>
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ==" crossorigin="anonymous" referrerpolicy="no-referrer"
+    />
+    <script src="https://kit.fontawesome.com/8f33fad190.js" crossorigin="anonymous"></script>
+    <!-- Sakhi UI Library -->
+    <link rel="stylesheet" href="https://sakhi-ui.netlify.app/Components/Card/card.css">
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="./../css/main.css">
+</head>
+
+<body>
+    <!-- navigation bar -->
+    <div class="navbar">
+        <div class="logo">
+            <a href="./../index.html"> <img src="/assets/images/nav_icons/logo.png" alt="logo" class="logo-img"></a>
+        </div>
+        <div class="search-bar">
+            <button class="search-bar-btn link-no-style">
+                <i class="fa fa-search"></i>
+            </button>
+            <input class="search-bar-input" type="text" placeholder="Type to search" value="">
+        </div>
+        <nav>
+            <ul id="MenuItems">
+                <li><a href="/pages/signin.html">Login</a></li>
+                <li><a href="/pages/products.html">Products</a></li>
+                <li>
+                    <a href="/pages/wishlist.html" class="badge-btn">
+                        <i class="fas fa-heart"></i>
+                        <span class="badge-count">6</span>
+                    </a>
+                </li>
+                <li>
+                    <a href="/pages/cart.html" class="badge-btn">
+                        <i class="fas fa-shopping-cart"></i>
+                        <span class="badge-count">3</span>
+                    </a>
+                </li>
+            </ul>
+        </nav>
+        <img src="/assets/images/nav_icons/menu.png" alt="menu" class="menu-icon" onClick="menutoggle()">
+    </div>
+
+    <!-- Login details-->
+    <div class="login-container pos-rel flex-grow display-flex-column">
+        <h1 class="login-form-title">Login</h1>
+        <form action="login-form" class="form-container display-flex-column">
+            <ul class="form-row display-flex-column">
+                <label for="Email address" class="email-label">Email address</label>
+                <input type="text" name="username" placeholder="username@gmail.com" class="input-box" required>
+            </ul>
+            <ul class="form-row form-row-pwd display-flex-column">
+                <label for="password" class="email-label email-label-pwd">Password</label>
+                <span class="password-eye-icon pos-rel">
+                    <input type="password" name="username" placeholder="**************" class="input-box " required/>
```

Comment:
1. You can use `type="email"` for email inputs. You can use it for better accessibility.
2. You have used for label `for="password"` attribute but haven't mentioned the corresponding id to the input element in both login and signup page.

Everything else looks good to me🙌

Reply:
thanks for pointing out this.. will make changes accordingly